docs/io.cozy.calendar.md

```diff
@@ -0,0 +1,54 @@
+[Table of contents](README.md#table-of-contents)
+    
+# Cozy Calendar doctype
+
+## `io.cozy.calendar`
+
+The `io.cozy.calendar` doctype is loosely based on the [iCalendar RFC](https://datatracker.ietf.org/doc/html/rfc5545), in that some of the attributes have been renamed for clarity. The attributes with a `?` are optional.
+
+- `start?` : {date} (example: `"1959-05-15"`)
+- `end?` : {date} (example: `"1959-05-15"`)
+- `label?` : {string}
+- `description?` : {string}
+- `location?` : {string}
+- `organiser?` : {string}
+- `status?` : {string} `CONFIRMED|CANCELLED`
+- `attendee?` : {object} Array of string with event attendees
```

Comment:
Is it an object or an array?

Reply:
Changed definition -> https://github.com/cozy/cozy-doctypes/commit/b5d47c756bc54d94408ffbe3ded2b8577354a7c3